test/test_raw_api.py

```diff
@@ -35,7 +36,7 @@ def _should_be_ok(self, filename):
         :param filename: unicode (or str) that follows the rules
         """
         print(u"Filename \"{0}\" should be OK".format(filename))
-        self.assertIsNone(self.raw_api.check_b2_filename(filename))
+        assert self.raw_api.check_b2_filename(filename) is None
```

Comment:
It's better to use `self.assert*`.  These methods are part of the unittest framework, and report errors nicely.

Feel free to add `assertIsNone` to TestBase.  We have some other things there that weren't supported in 2.6.